### PR TITLE
kci_doc: Fix buster-btrfs build documentation

### PR DESCRIPTION
**Pull request by mgalka**
This commit fixes build instructions for buster-btrfs rootfs. Moves
build time dependencies to the build script and cleans up some
unnecessary files.

Signed-off-by: Michal Galka <michal.galka@collabora.com>

### CODE REVIEW

File: doc/kci_testsuite.md [hunk 1]
```diff
@@ -88,34 +88,20 @@ For this example, we will be using the `btrfs-progs` test-suite.
 1. Let's first modify `rootfs-configs.yaml` to add `buster-btrfs` with appropriate entries.
 
 ```
+  buster-btrfs:
```
Comment (gctucker) [inline doc/kci_testsuite.md:91]: Well, please clean up.

**Review by gctucker — CHANGES_REQUESTED** [review]

File: doc/kci_testsuite.md (hunk 1 above)
Comment (mgalka) [reply doc/kci_testsuite.md]: Done

File: doc/kci_testsuite.md [hunk 2]
```diff
@@ -125,11 +110,18 @@ We will now look into last three config entries, namely: `extra_packages`, `scri
 ```
 $ cat jenkins/debian/debos/script/buster-btrfs.sh
 
-git clone --depth=1 https://github.com/kdave/btrfs-progs.git /btrfs-progs
+#!/bin/bash
+
+BUILD_DEPS="pkg-config git make autoconf automake gcc \
+make libblkid-dev zlib1g-dev liblzo2-dev libzstd-dev e2fslibs-dev python3-setuptools python3.7-dev"
+
+apt-get update -y && apt-get install -y $BUILD_DEPS
```
Comment (gctucker) [inline doc/kci_testsuite.md:118]:
```suggestion
apt-get update && apt-get install -y $BUILD_DEPS
```

File: doc/kci_testsuite.md [hunk 3]
```diff
@@ -125,11 +110,18 @@ We will now look into last three config entries, namely: `extra_packages`, `scri
...
+GIT_SSL_NO_VERIFY=1 git clone --depth=1 https://github.com/kdave/btrfs-progs.git /btrfs-progs
 cd /btrfs-progs
-./autogen.sh && ./configure
-make
-make install
+./autogen.sh && ./configure --disable-documentation
```
Comment (gctucker) [inline doc/kci_testsuite.md:121]: Maybe add `set -e` at the top of the script and drop the `&&` to put only one command on each line.

**Review by gctucker — COMMENTED** [review]
Thanks, looks good.  Only thing is that `kci_doc` in the commit subject looks like a command line tool like `kci_build` but it's not.  I see a previous commit from @Lakshmipathi also did something like that, but `doc:` would be better.

**Comment by mgalka** [comment]
> Thanks, looks good. Only thing is that `kci_doc` in the commit subject looks like a command line tool like `kci_build` but it's not. I see a previous commit from @Lakshmipathi also did something like that, but `doc:` would be better.

That's exactly why I started the commit message with `kci_doc`. Changed it to `doc`.

**Review by gctucker — APPROVED** [review]

**Pull request merged**